saleor/graphql/translations/mutations.py

```diff
@@ -80,7 +82,17 @@ def clean_node_id(cls, **data):
             )
 
         node_id = data["id"]
-        node_type, node_pk = graphene.Node.from_global_id(node_id)
+        try:
+            node_type, node_pk = from_global_id_or_error(node_id)
+        except GraphQLError:
+            raise ValidationError(
+                {
+                    "id": ValidationError(
+                        "Invalid ID has been provided.",
+                        code="not_found",
```

Comment:
I think we have `not_found` constant already in the codebase?

Reply:
TBH I don't really think that "not_found" should be used here.. it is raised when ID is invalid format not when its not found

Reply:
I will change it to `code=TranslationErrorCode.INVALID`